### ISSUE TITLE
Neutrophil migratory defect

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5867]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
see https://pubmed.ncbi.nlm.nih.gov/26224645/

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
This is https://hpo.jax.org/app/browse/term/HP:0040238
I will add synoynm.
